sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -463,7 +463,7 @@ export class KeyClient {
   public async importKey(
     name: string,
     key: JsonWebKey,
-    options: ImportKeyOptions
+    options?: ImportKeyOptions
```

Comment:
This was originally made required because of aligning with the C# change:

"Remove ImportKey overload that does NOT take ImportKeyOptions"

I'm fine making it optional, but was trying to align with the change request.

Reply:
On second thought, everything in ImportKeyOptions is optional, so it doesn't make sense to require this.